From 3dba83ce7c6a70e7262aecc6f4f73d96b7e8f45d Mon Sep 17 00:00:00 2001
From: Kevin Chen <kevin_chen@aspeedtech.com>
Date: Wed, 27 Aug 2025 17:08:29 +0800
Subject: [PATCH 14/32] configs: ibex-ast2700: Update bootloader-fit load
 address

Adjusts the system load address from 0xa3000000 to 0x83000000 for the
AST2700 target configuration. This change aligns with updated memory
layout requirements or optimizations for the platform.

Change-Id: I26bd3f7a2bbc1a07b9854a8f61b536cd61412a72
Signed-off-by: Kevin Chen <kevin_chen@aspeedtech.com>
(cherry picked from commit 4cb59918d6bc3c818e51fa6ef67adabfd17e5561)
Signed-off-by: Vasanthakumar K <vasanthakumark@ami.com>
---
 configs/ibex-ast2700_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/ibex-ast2700_defconfig b/configs/ibex-ast2700_defconfig
index a1d5d51f95..2bafebc355 100644
--- a/configs/ibex-ast2700_defconfig
+++ b/configs/ibex-ast2700_defconfig
@@ -18,7 +18,7 @@ CONFIG_SPL_MMC=y
 CONFIG_SPL_SIZE_LIMIT=0x21e00
 CONFIG_SPL=y
 CONFIG_SYS_MEM_TOP_HIDE=0x10000000
-CONFIG_SYS_LOAD_ADDR=0xa3000000
+CONFIG_SYS_LOAD_ADDR=0x83000000
 CONFIG_BUILD_TARGET=""
 CONFIG_TARGET_ASPEED_AST2700=y
 CONFIG_FMC_SPECIFIC_OPTIONS=y
-- 
2.34.1

